styles.css

```diff
@@ -0,0 +1,43 @@
+html {font-family: 'Poppins', sans-serif;
+
+}
+a 
+{
+  color: black
+}
+body {
+    background-color: #3845;
+    color: rgba(0, 0, 0, 0.445)
+  }
+h3{
+  font-style: italic;
+}
+h4{
+  padding-left: 20px;
+  text-align: center;
+  line-height: 1.5;
+  display: inline-block;
+  vertical-align: middle;
+}
+  
+  h1, h2, h3 {
+    color: black;
+    text-align: center;
+    
+  }
+  h4, .list {
+    color: black;
+    font-kerning: .200;
```

Comment:
This is not a valid value for [`font-kerning`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-kerning):
```suggestion
```